app/models/Attempt.scala

```diff
@@ -79,9 +88,7 @@ object Attempt {
     */
   def successfulAttempts[A](attempts: List[Attempt[A]])(implicit ec: ExecutionContext): Attempt[List[A]] = {
     Attempt.Async.Right {
-      Future.sequence(attempts.map { attempt =>
-        attempt.fold(_ => None, a => Some(a))
-      }).map(_.flatten)
+      Future.traverse(attempts)(_.fold(_ => None, a => Some(a))).map(_.flatten)
```

Comment:
That might be just me but I found this line quite hard to read with all those parenthesis. 🙈